pkgs/applications/editors/emacs/elisp-packages/manual-packages/straight/default.nix

```
@@ -0,0 +1,27 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  lib,
+}:
+
+melpaBuild {
+  pname = "straight";
+  version = "0-unstable-2025-01-25";
+
+  src = fetchFromGitHub {
+    owner = "radian-software";
+    repo = "straight.el";
+    rev = "33fb4695066781c634ff1c3c81ba96e880deccf7";
+    hash = "sha256-3NPVLTn0ka0RvSLXW9gDKam3xajp62/mLupc8uyatzo=";
+  };
+
```

It fails to build:
```console
> Straight unable to find required executable: "git"
```

Please at least make sure the build succeeds before creating a PR.  You can use this command in the Nixpkgs repo:

```bash
nix build -Lv --include nixpkgs=$PWD --file . emacs.pkgs.straight
```

```suggestion
  nativeBuildInputs = [ git ];
```